src/TemplateBuilder.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Staffim\SpreadsheetBuilder;
+
+use PhpOffice\PhpSpreadsheet\Reader;
+use PhpOffice\PhpSpreadsheet\RichText\RichText;
+use PhpOffice\PhpSpreadsheet\Writer;
+use PhpOffice\PhpSpreadsheet\Spreadsheet;
+
+class TemplateBuilder
```

Comment:
need to rename `AbstractWorksheetBuilder` to `AbstractTableWorksheetBuilder`, add `AbstractWorksheetBuilder` class for all builders

Reply:
Added `AbstractBuilder` class. I think `AbstractWorksheet` have another usage context and cannot be inherit from one class